.github/actions/functest/action.yml

```diff
@@ -26,8 +26,8 @@ inputs:
     description: For auto-determining the default cross-prefix (e.g. native | cross )
     default: "native"
   opt:
-    description: opt flag to pass to test script
-    default: "true"
+    description: Determine for building opt/non-opt or both of the binaries (e.g. all | opt | no_opt)
```

Comment:
Thanks for the check, I forgot to run no-opt code when `tests bench --opt all`... 🤦‍♂️

Reply:
Note that when `--opt=all`, and if `--output <file>` is provided, the current tests script will benchmark both opt/no-opt implementations, but only write the output result to the output file.

Since we haven't yet decided how to output to the same file for both implementations. I've left a comment in the script about this, this should be sufficient for now.